.github/workflows/run-tests.yml

```diff
@@ -487,12 +488,24 @@ jobs:
           annotate_only: true
           skip_annotations: true
 
+      - name: Get job ID from workflow run
+        id: get_job_id
+        run: |
+          set -e          
```

Comment:
```suggestion
          set -euo pipefail
```

Reply:
I think this may be easier to obtain with the `gh` cli: https://stackoverflow.com/questions/71240338/obtain-job-id-from-a-workflow-run-using-contexts.